util/util.go

```diff
@@ -32,7 +32,7 @@ func IsValidURL(toTest string) bool {
 //a given client / URL(string) / bearerToken(string)/ retries count (int)
 //and returns true if response code is 2xx.
 func TestHTTPConnection(testClient rest.HTTPClient,
-	URL string, bearerToken string, retries uint) (successful bool, body *[]byte, err error) {
+	URL string, bearerToken string, retries uint, verbose bool) (successful bool, body *[]byte, err error) {
```

Comment:
Is this something we want to make configurable by the user?

Reply:
(the verbose option, that is)

Reply:
At some point, I'd like to eventually tie that to a debug mode configurable by the client.